examples/chip-tool/py_matter_chip_tool_adapter/matter_chip_tool_adapter/encoder.py

```diff
@@ -242,11 +324,18 @@ def __maybe_add(self, rv, value, name):
         rv += f'"{name}":"{value}"'
         return rv
 
-    def __encode_value(self, value):
+    def __encode_value(self, command_name, argument_name, value):
+        value = self.__encode_wildcards(command_name, argument_name, value)
         value = self.__encode_octet_strings(value)
         value = self.__lower_camel_case_member_fields(value)
         return self.__convert_to_json_string(value)
 
+    def __encode_wildcards(self, command_name, argument_name, value):
+        if not value == "*" or command_name not in _ANY_COMMANDS_LIST or argument_name not in _ANY_COMMANDS_LIST_ARGUMENTS_WITH_WILDCARDS:
```

Comment:
nit: reversed conditions (i.e. using `not`) see a bit harder to parse. Instead of `not ==` maybe `!=` is correct and shorter logic

```python
if value != '*':
   return value

# maybe a whildcard
if command_name in _ANY_COMMANDS_LIST and argument_name in _ANY_COMMANDS_LIST_ARGUMENTS_WITH_WILDCARDS:
    # translate * to wildcard constant
    return 0xFFFFFFFF
    
 return value
 ```

Reply:
Updated. Thanks !